collections/Media/IFrame/PlayView.tsx

```diff
@@ -0,0 +1,20 @@
+import { useIframeStore } from '@/collections/Media/IFrame/store'
+
+const PlayView = () => {
+  const link = useIframeStore(state => state.link)
+  const width = useIframeStore(state => state.width)
+  const height = useIframeStore(state => state.height)
+  return (
+    <div className="text-center">
+      <iframe
+        className="relative w-full overflow-hidden pr-12"
+        width={width}
+        height={height}
```

Comment:
I'm okay specifying the `height` but please do not set the `width`. The width should be 100% (as already specified in the `className`

Reply:
Alright, I removed the `width`.